### DISCUSSION TITLE
Is there a way of providing color palletes?

### DISCUSSION THREAD

**Discussion by D4yvid** [slint-ui/slint#6342] [Q&A] [answered]
I want to know if is there any way of changing the color pallete of the theme itself, for example:

```slint
Pallete.foreground = #ffffff; // "Error: Assignment on a output property"
```

Is this possible? Like, i was trying to do something like MaterialYou-style of Android, that generates a pallete using wallpaper colors

**Comment by ghost** [comment, marked as answer]
Hi,

that is currently not possible because of the properties on `Property` are all `out` properties (readonly). We plan to have more customization capabilities in the future. There are some related issue like #5392.

**Reply by D4yvid** [reply]
Okay! Thanks for the reply :)
